docs/howto/respectful-collaboration.md

```diff
@@ -0,0 +1,6 @@
+## How to build respectful collaborative environments on Github
+
+WARNING: This is an incomplete page which will be gradually populated with content.
+
+1. _Consider a Code of Conduct for your project_.
```

Comment:
Also make sure the CoC is _actionable_. It it gives a contact point (most CoC do, as they should since people must be able to report violations if the CoC is to be enforceable at all), make sure the contact point actually exists and is ready to receive report.

Reply:
Added to the howto.